detectable/src/test/java/com/synopsys/integration/detectable/detectables/go/functional/GoModTestUtil.java

```diff
@@ -0,0 +1,36 @@
+package com.synopsys.integration.detectable.detectables.go.functional;
+
+import com.synopsys.integration.executable.ExecutableOutput;
+
+public class GoModTestUtil {
+
+    private GoModTestUtil() {}
+
+    public static ExecutableOutput createGoModWhyOutputWithPrefix(String modulePrefix) {
```

Comment:
I'm torn on this. I don't mind tests repeating data but this makes your tests shorter. Would it make sense to just have one test that uses a mix of both prefixes you want to test?

Reply:
We only run one extraction per Detectable test. So I wouldn’t be able to test the different config options. Only one

Reply:
A `MultiRunDetectableTest` might be something worth noodling over.

Reply:
The data could be included in both places while looking similar.
The tests already share multiple resource files. I thought it would make sense to do the same here. 

Ideally I think we would have the ability to easily load FTL files from resource files as well. 

For now, I'll remove the utility and have similar, but different data in both tests.